test/e2e/suite/issuers/acme/certificate/webhook.go

```diff
@@ -197,12 +198,13 @@ func listOwnedOrders(cl versioned.Interface, owner *v1.Certificate) ([]*cmacme.O
 	}
 
 	var owned []*cmacme.Order
-	for _, o := range l.Items {
+	for i := range l.Items {
+		o := &l.Items[i]
 		v, ok := o.Annotations[v1.CertificateNameKey]
 		if !ok || v != owner.Name {
 			continue
 		}
-		owned = append(owned, &o)
+		owned = append(owned, o.DeepCopy())
```

Comment:
As discussed verbally, DeepCopy is overkill, a shallow copy is enough to prevent the slice from being freed.